### PR TITLE
Updated Complete Drawpieces

### PR DESCRIPTION
**Pull request by IlianaP**
I am sorry for creating a new branch, but my terminal was acting silly and I spent an hour trying to fix it, but the db wouldn't update. 
So, this is the new branch, with the two comments Erin made in the previous one (drawpieces) fixed. :)

### CODE REVIEW

**Comment by wdehaes** [comment]
@IlianaP what is the use of the "wp" and "bp" classes? I assume they are for pieces, but do you make use of them somewhere?

File: app/views/games/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,70 @@
...
+<p>Game name: <%= @game.name %></p> 
+
+<% array = ["A","B","C","D","E","F","G","H"] %> 
+<% unicodefig = ["&#9820;","&#9822;", "&#9821;", "&#9819;", "&#9818;", "&#9821;", "&#9822;", "&#9820;"] %>
+<table id = "ChessBoard"> 
+<% 8.times.each do |number| %> 
+	<% if number% 2 ==0 %>  
+		<tr> 
+			<% if number == 6 %> 
+				<% 8.times.each do |square| %>
```
Comment (lasley) [inline app/views/games/show.html.erb:12]: Seems like we're duplicating a lot of effort here. Why not just create an `additional_class` variable or something, then assign the ` `,`wp`or`bp` to it, then just interpolate with the static white/black class?

Also, what do the `wp` and `bp` mean? Kind of ambiguous naming IMO

Comment (IlianaP) [reply app/views/games/show.html.erb:12]: Lol, it's wp= white player and bp = black player, I used those for the css classes

**Comment by IlianaP** [comment]
@wdehaes 
wp= white player and bp = black player, I used those for the css classes. You can find them defined in the css and used in the show.html file.

File: app/views/games/show.html.erb (hunk 1 above)
Comment (IlianaP) [reply app/views/games/show.html.erb:12]: @lasley so, do I still need to change anything? I am not sure I quite get what you mean with the additional_class variable.

**Review by lasley — APPROVED** [review]

**Comment by erinmorrissey** [comment]
Merging.

**Pull request merged**